src/framework/anim/controller/anim-controller.js

```diff
@@ -450,7 +450,7 @@ class AnimController {
         const path = pathString.split('.');
         let state = this._findState(path[0]);
         if (!state) {
-            state = new AnimState(this, path[0], 1.0);
+            state = new AnimState(this, path[0], speed);
```

Comment:
See constructor for AnimState https://github.com/playcanvas/engine/blob/8fd5a033065fb4dfcd0fad4b1ecbdf25591e0f45/src/framework/anim/controller/anim-state.js#L40